bulletin/index.html

```diff
@@ -203,6 +203,53 @@ <h2 class="featured-article__title"> Engineering Robust Learning Environments Th
 	  </div>
 	</section>
 
+	<section class="mention">
+		<div class="mention-container mention-container__wrapper">
+			<div class="mention-heading">
+				<div> 
+					<h2 class="mention-heading__title">Comments & Acknowledgments</h2>
+				</div >
+				<div>
+					<p class="mention-heading__description">We recognize the contributions and heartfelt comments of the people, companies, and communities that characterize MACILE. They are invaluable. Without them, improving the  program and meeting its objectives would be difficult.</p>
+				</div>
```

Comment:
Same here, is the `div` wrapping `mention-heading__description` needed?

Reply:
Same as my previous response.